src/main/java/seedu/address/logic/commands/JokeCommand.java

```diff
@@ -22,4 +22,9 @@ public JokeCommand(JokeList jokeList) {
     public CommandResult execute(ItemModel model) throws CommandException {
         return new CommandResult(jokeList.getJoke());
     }
+
+    @Override
+    public String getCommandWord() {
+        return COMMAND_WORD;
+    }
```

Comment:
Just feel that this method should not be implemented as it serves no purpose. In fact the method `getCommandWord()` should only be implemented in the UndoableCommand class since that is where it is used.

Reply:
I'll change that in a new PR, since this is not the only place I need to change it :)